lib/apiservers/engine/backends/cache/image_cache.go

```diff
@@ -155,6 +155,12 @@ func (ic *ICache) Get(idOrRef string) (*metadata.ImageConfig, error) {
 		return copyImageConfig(config), nil
 	}
 
+	// let's try to get the image ID from the repocache just in case we have a layer ID
+	// see https://github.com/vmware/vic/issues/4378#issuecomment-289070799
+	if id := RepositoryCache().GetImageID(strings.Split(idOrRef, ":")[0]); id != "" {
```

Comment:
This is from Zac's newly merged PR. I might have done something wrong when I rebase my code base.